.drone.yml

```diff
@@ -0,0 +1,18 @@
+build:
+  image: gregway/gbm_immune_validation:latest
+  commands:
+    - bash run_pipeline.sh
+
+    - git config user.email "gregory.way@gmail.com"
+    - git config user.name "gwaygenomics"
+    - git config --global push.default simple
+    - git remote set-url origin https://github.com/greenelab/gbm_immune_validation
+
+    - git checkout master
+    - git add *
+    - git commit -a -m "Drone Build [skip ci] [ci skip]"
```

Comment:
I only use both in some cases because different services use different versions. Drone handles both - 

Instruct Drone to skip builds by by including any combination of ci and skip wrapped in square brackets in your commit message. Examples: [skip CI] [ci skip]